resources/views/components/input/text.blade.php

```diff
@@ -4,7 +4,7 @@
     <input
         id="{{$id}}"
         class="validate @error($id) invalid @enderror"
-        value="{{old($id) ?? $attributes->get('value')}}"
```

Comment:
The old value is needed when the controller redirects back with validation errors, it keeps the previously filled values. Please check if you still have that behavior.

Reply:
It seems okay to me, but not sure if I'm testing the right behavior (I know for fact that some validation is done on the client side too).

How would you recommend testing this?

Reply:
I've now tested it by entering a non-numeric value to closed university semester grades (those are only checked by the validator). It worked for me too.

Reply:
Ah, I see, sorry.